share/examples/tests/tests/plain/printf_test.c

```diff
@@ -62,7 +62,7 @@ snprintf__two_formatters(void)
 static void
 snprintf__overflow(void)
 {
-	char buffer[10];
+	char buffer[];
```

Comment:
This is rather nonsensical. You have to allocate _some_ sort of buffer to print to, you cannot just specify a flexible array. Since this particular example is meant to demonstrate a `snprintf` truncation, the buffer size should stay as it is. Of course, this results in a compiler warning, but in this very specific case it can be completely and safely ignored.